Apple(iOS) libc::sockaddr_in does not impl Debug trait

Apple(iOS) `libc::sockaddr_in` does not impl Debug trait, but tun2 defines `SockAddr` struct with Debug trait implmentation.

https://github.com/tun2proxy/rust-tun/blob/v2/src/platform/posix/sockaddr.rs#L20

```
   Compiling tun2 v1.2.8
error[E0277]: `sockaddr_in` doesn't implement `Debug`
  --> .../.cargo/registry/src/index.crates.io-6f17d22bba15001f/tun2-1.2.8/src/platform/posix/sockaddr.rs:21:21
   |
20 | #[derive(Copy, Clone, Debug)]
   |                       ----- in this derive macro expansion
21 | pub struct SockAddr(sockaddr_in);
   |                     ^^^^^^^^^^^ `sockaddr_in` cannot be formatted using `{:?}` because it doesn't implement `Debug`
   |
   = help: the trait `Debug` is not implemented for `sockaddr_in`, which is required by `&sockaddr_in: Debug`
   = note: this error originates in the derive macro `Debug` (in Nightly builds, run with -Z macro-backtrace for more info)
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -15,7 +15,7 @@ crate-type = ["staticlib", "cdylib", "lib"]
 bytes = { version = "1" }
 cfg-if = "1.0"
 futures-core = { version = "0.3", optional = true }
-libc = "0.2"
+libc = { version="0.2", features=["extra_traits"] }
 log = "0.4"
 thiserror = "1"
 tokio = { version = "1", features = [

